styles/designs/carnival/parts/_chapterintro-shapes.scss

```diff
@@ -34,7 +34,7 @@
                         map-merge($ChapterIntro__Container--Text, (
                             _components: (
                                 $H2TextHeader--ChapterIntro,
-                                $Note__OrderedList--MarginLeftBottom, //added as e tamporary solution before the lists cleaning issue will be resolved
+                                $OrderedList--FirstLevel--MarginBottom, // present only in pl-u-physics (needs bottom margin)
```

Comment:
I have left this list in chapter intro and  chapter intro body shapes instead of creating a separate list shape just for one list. It is needed only in pl-u-physics, where without the bottom margin the list in intro text will look odd: 
![image](https://user-images.githubusercontent.com/55203598/184857527-538ff0eb-ae32-4a2f-886b-78d1b819667f.png)